### PR TITLE
Created OPI for Wish Jaws but a couple of PVs are missing

### PR DESCRIPTION
**Pull request by daryakoskeroglu**
### Description of work

Created OPI for Wish Jaws but a couple of PVs are missing.

PVs that are connected are Vertical and Horizontal Gap and their setpoints.

The OPI is created based on the VI:  (located here: C:\LabVIEW Modules\Instruments\WISH\WISH Jaws).)

WISH Front Panel

There is also a Setup Dialogue to set gaps which is written to the text file on LabVIEW. This gaps could be set via editing the text file directly which then can be seen in the Front Panel. However, WISH scientists do not use neither Setup Dialogue nor text file.

Also they added that:

"The jaws are jittering a bit and sometimes to get the right value you need to send a value that is a bit off and repeat the operation several times so actually we just set values in the VI and drive."

### Things needs to be done 

1. Populate "Experiment Type" menu, possibly create an enum PV for this.
2. Connect Resolution menu, Dead Band, Position error and Activity LEDs , and set and start/stop buttons.

### Ticket

[*Link to Ticket*](https://github.com/ISISComputingGroup/IBEX/issues/6033)
---

#### Code Review

- [ ] Is the code of an acceptable quality?
- [ ] Do the changes function as described and is it robust?
- [ ] Is there associated PR for the [release notes](https://github.com/ISISComputingGroup/IBEX/blob/master/release_notes/ReleaseNotes_Upcoming.md)?

### Final Steps
- [ ] Reviewer has also merged the [release notes](https://github.com/ISISComputingGroup/IBEX/blob/master/release_notes/ReleaseNotes_Upcoming.md)

### CODE REVIEW

**Review by JamesKingWork — CHANGES_REQUESTED** [review]
The OPI is simple and matches the VI which is good. We have a nice more_detail_manager.opi which provides a graphical view and more control over the jaws, can we add a button that opens this more_details_manager with the wish jaws. An example of this is in the nimrod_manager.opi and gem_manager.opi with the button "Jaws Manager". It would probably be good to place this in the same grouping container where the gaps can be set as the information in the more_detail_manager.opi is most connected to those gaps.

**Comment by Adam-Szw** [comment]
The OPI looks good and passes `check_opi_format` test. The _Jaws Manager_ button from James' suggestion was added. Just a small change request:
- Please add entry for the OPI to `opi_info.xml` so that this OPI can be selected from the client.
I expect that functionality of the OPI and/or IOC will be reviewed here https://github.com/ISISComputingGroup/IBEX/issues/6917 so I'm leaving PV connectivity out here.

**Comment by daryakoskeroglu** [comment]
> The OPI looks good and passes `check_opi_format` test. The _Jaws Manager_ button from James' suggestion was added. Just a small change request:
> 
> * Please add entry for the OPI to `opi_info.xml` so that this OPI can be selected from the client.
>   I expect that functionality of the OPI and/or IOC will be reviewed here [WISH: Connect PVs to WISH Jaws OPI IBEX#6917](https://github.com/ISISComputingGroup/IBEX/issues/6917) so I'm leaving PV connectivity out here.

Thanks for the review, I've updated opi_info.xml

File: base/uk.ac.stfc.isis.ibex.opis/resources/opi_info.xml [hunk 1]
```diff
@@ -2691,6 +2691,18 @@
           </macro>
         </macros>
       </value>
+    </entry>
+     <entry>
+      <key>WISH Jaws</key>
+      <value>
+        <type>JAWS</type>
+        <path>jaws/jaws_managers/WISH_Jaws.opi</path>
```
Comment (Adam-Szw) [inline base/uk.ac.stfc.isis.ibex.opis/resources/opi_info.xml:2699]: There was a typo here, I fixed it quickly

**Review by Adam-Szw — APPROVED** [review]

**Review by Adam-Szw — APPROVED** [review]

**Review by Adam-Szw — APPROVED** [review]

**Pull request merged**